### PR TITLE
Fbp webinspector

### PR DESCRIPTION
**Pull request by barbieri**
first try, missing html/js/css.

Could you guys take a look and see with some of the designers if that could be useful for them? I'll do some static HTML/JS/CSS that will render a table with nodes and connected ports, with that I plan to put the last value and maybe plot a graph of values-over-time.

This is to address #1585

### CODE REVIEW

File: src/bin/sol-fbp-runner/main.c [hunk 1]
```diff
@@ -120,7 +146,31 @@ parse_args(int argc, char *argv[])
...
+            break;
+#endif
+#ifdef WEB_INSPECTOR
+        case 'W':
+            if (optarg) {
+                unsigned long v;
+                char *endptr;
+
+                errno = 0;
+                v = strtoul(optarg, &endptr, 10);
```
Comment (ceolin) [inline src/bin/sol-fbp-runner/main.c]: we have `sol_util_strtoul`  that is exported now.

File: src/bin/sol-fbp-runner/Makefile [hunk 2]
```diff
@@ -3,4 +3,11 @@ bin-sol-fbp-runner-$(FBP_RUNNER) := main.c runner.c
 
 ifeq (y,$(INSPECTOR))
 bin-sol-fbp-runner-$(FBP_RUNNER) += inspector.c
+
+ifeq (y,$(HTTP_SERVER))
```
Comment (glima) [inline src/bin/sol-fbp-runner/Makefile]: I wonder if we don't want an *and ``SOL_FLOW_INSPECTOR_ENABLED``* here

File: src/bin/sol-fbp-runner/main.c (hunk 1 above)
Comment (barbieri) [inline src/bin/sol-fbp-runner/main.c]: that's not useful here, the regular version will do all right

File: src/bin/sol-fbp-runner/Makefile (hunk 2 above)
Comment (barbieri) [inline src/bin/sol-fbp-runner/Makefile]: it's nested under `ifeq (y,$(INSPECTOR))` that is baaed on the same config value that results in `SOL_FLOW_INSPECTOR_ENABLED`

File: src/bin/sol-fbp-runner/web-inspector.c [hunk 3]
```diff
@@ -0,0 +1,1396 @@
...
+        if (r < 0)
+            return r;
+
+        r = sol_json_serialize_string(buf, kv->value);
+        if (r < 0)
+            return r;
+
+        r = sol_buffer_append_char(buf, ']');
+        if (r < 0)
+            return r;
```
Comment (ceolin) [inline src/bin/sol-fbp-runner/web-inspector.c]: Don't you think it's worth log this error ? Don't need to be here, can be in the top leve function, e. g. `web_inspector_will_deliver_packet`. As far as i could see the error is not spotted.

Comment (barbieri) [inline src/bin/sol-fbp-runner/web-inspector.c]: will do the logging for these errors.

**Comment by barbieri** [comment]
New version at #1802

**Pull request closed without merging**